src/credentials/wizards/createProfile.ts

```diff
@@ -43,7 +44,13 @@ export class CreateProfileWizard extends Wizard<CreateProfileState> {
     public constructor(profiles: { readonly [name: string]: Profile }, template: ProfileTemplateProvider) {
         super()
 
-        this.form.name.bindPrompter(() => createProfileNamePrompter(profiles))
+        if (Object.keys(profiles).length === 0) {
+            // Skip the "Enter Profile" step and use "default" profile name.
+            // This reduces friction for the common case.
+            this.form.name.bindPrompter(() => new PassthroughPrompter('default'))
```

Comment:
@JadenSimon is there another way to force `name = 'default'`?

Reply:
Yeah there's a declarative and imperative way:

Declarative:
```ts
    super({ initState: { name: Object.keys(profiles).length === 0 ? 'default' : undefined } })
    this.form.name.bindPrompter(() => createProfileNamePrompter(profiles))
```

Imperative:
```ts
     if (Object.keys(profiles).length === 0) {
         this.form.name.setDefault('default')
     } else {
         this.form.name.bindPrompter(() => createProfileNamePrompter(profiles))
     }
```

Haven't really decided on what the 'best' way is. So far devs seem to prefer the imperative approach so probably that one for simple scenarios.